@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -436,8 +438,9 @@ export class TextElementsRenderer {
 
         this.updateGlyphDebugMesh();
 
-        for (const textRenderer of this.m_textRenderers) {
-            textRenderer.textCanvas.render(camera);
+        for (const screenSpaceRenderer of this.m_screenSpaceRenderers) {
+            screenSpaceRenderer.poiRenderer?.render(camera);
+            screenSpaceRenderer.textCanvas?.render(camera);
```

Comment:
If you render all icons first we can't put an icon with higher render order on top of text with lower render order.

Reply:
I added another commit, now supporting interleaved rendering